rfc8446.xml

```diff
@@ -2523,7 +2524,7 @@ included in the transcript hash.</t>
   <t hangText='Certificate:'>
   The certificate to be used for authentication, and any
 supporting certificates in the chain. Note that certificate-based
-client authentication is not available in PSK (including 0-RTT) flows.</t>
+client authentication is not available in PSK (including 0-RTT) handshake flows.</t>
```

Comment:
maybe "PSK handshake flows (including 0-RTT)"

Reply:
That's better, yes.

Reply:
Done.